core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2.scala

```diff
@@ -42,6 +47,24 @@ trait BillingApiServiceV2 extends UserInfoDirectives {
               }
             }
         } ~
+        pathPrefix("spendReport") {
+          pathEndOrSingleSlash {
+            get {
+              parameters("startDate".as[String], "endDate".as[String]) { (startDate, endDate) =>
+                complete {
+                  spendReportingConstructor(userInfo).getSpendForBillingProject(
+                    RawlsBillingProjectName(projectId),
+                    DateTime.parse(startDate),
```

Comment:
What sort of error will we see if the startDate/endDate cannot be parsed?

Reply:
And is it worth having a test for that case?

Reply:
Yeah, there definitely should be a test for this